designsystem/src/commonMain/kotlin/org/hisp/dhis/mobile/ui/designsystem/component/BasicTextInput.kt

```diff
@@ -54,28 +57,33 @@ internal fun BasicTextInput(
 
     var deleteButtonIsVisible by remember { mutableStateOf(!inputText.isNullOrEmpty() && state != InputShellState.DISABLED) }
     val focusManager = LocalFocusManager.current
+
+    val button: (@Composable () -> Unit)?
```

Comment:
Why don't initialize button as null instead of adding the else branch?

Reply:
good catch, I've just corrected it